src/BrazeKit-dev.js

```diff
@@ -760,7 +760,7 @@ var constructor = function () {
                 finishBrazeInitialization(forwarderSettings);
             } else {
                 if (!braze.initialize(forwarderSettings.apiKey, options)) {
-                    return 'Failed to initialize: ' + name;
+                    return 'Failed to initialize: ' + name;corrects 
```

Comment:
Why is this `corrects;` statement just dangling here?

Reply:
I don't remember. Removed.